src/Kushy/SymbolLoader.cs

```diff
@@ -285,11 +323,14 @@ private async Task<GlobalState> AddReferencedDatabasesAsync(GlobalState globals,
                 {
                     // look for existing database of this name
                     var db = cluster.Databases.FirstOrDefault(m => m.Name == dbRef.Database);
-
+                    if(db == null)
```

Comment:
Is this supposed to succeed when the prior line did not?

Reply:
removed this.